Blog_part_2.md

```diff
@@ -0,0 +1,307 @@
+
+#### Deploy Kubernetes Cluster
+
+First create the [Google Container Engine](http://cloud.google.com/container-engine) cluster using the `gcloud` command as shown below. 
+
+**Note:** This command defaults to creating a three node Kubernetes cluster (not counting the master) using the `n1-standard-1` machine type. Refer to the [`gcloud container clusters create`](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create) documentation information on specifying a different cluster configuration.
+
+    $ gcloud container clusters create CLUSTER-NAME
+
+If you do not have a default Google Cloud Project Id set you can append the `--project=` argument to all of your gcloud commands like so:
+
+    $ gcloud container clusters create CLUSTER-NAME --project=PROJECT-ID
+
+If you do not know if you have a project set run the following command:
+
+    $ glcoud config list
+
+And you should see an output like so:
+
+    [compute]
+    zone = us-central1-a
+    [core]
+    account = email.address@somedomain.com
+    project = your-google-cloud-project-id
+
+I recommend setting both your project and compute zone. You can find the directions for both in the Google Cloud Tools Documentation.
+
+Now let's take a look at the type of machines we can run our container nodes on. Run the following command to get a list:
+
+    $ gcloud compute machine-types list
+
+You can also visit the [Google Compute Machine Type Documenation](https://cloud.google.com/compute/docs/machine-types) to learn more.
+
+In our case we're going to use some higher cpu machines as CPU is often the limiting factor when running locust. The command below uses the `n1-highcpu-8` machine type. This is a High-CPU machine type with 8 virtual CPUs and 7.20 GB of memory. Obviously the more powerful machines you use the more expensive they are. Please use what is comfortable for you. Just note that if you notice the locust system hitting failures you may need to either increase the CPU or Memory of your machines or add more machines.
+
+Since we have 7 workers nodes with 1 controller we need to let container engine know by passing in the `--num-nodes` parameter with a value of 8 in our case. The command looks like so:
+
+    $ gcloud container clusters create CLUSTER-NAME --machine-type=n1-highcpu-8 --num-nodes=8
+
+Let's call our cluster here `locust-cluster` so the exact command we will run is:
+
+    $ gcloud container clusters create locust-cluster --machine-type=n1-highcpu-8 --num-nodes=8
+
+Now let's make the cluster our default cluster in this project by adding it to our gcloud config with the following comand:
+
+    $ gcloud config set container/cluster locust-cluster
+
+You can run `gcloud config list` again to confirm the following entry:
+
+    [container]
+    cluster = locust-cluster
+
+Get credential for your cluster.
+
+    $ gcloud container clusters get-credentials locust-cluster
```

Comment:
Perhaps add a ``kubectl get pods`` command to check that authentication is set up successfully?